backbone/src/main/java/org/researchstack/backbone/result/logger/DataLoggerFileWriterThread.java

```diff
@@ -178,6 +179,10 @@ private void writeCanceledFromThreadToMainThread() {
             @Override
             public void run() {
                 thread.quit();
+                boolean success = file.delete();
+                if (!success) {
+                    Log.d(getClass().getSimpleName(), "Failed to delete file " + file.toString());
```

Comment:
We seem to have a lot of inconsistency around logging.  Backbone has a LogExt class that is used elsewhere.  I don't really have a preference, but consistency would be nice.